styles.css

```diff
@@ -0,0 +1,44 @@
+body {
+  margin: 0;
+  height: 100vh;
+  overflow: hidden;
+}
+
+h1 {
+  font-size: 36px;
+  color: #333;
+  text-align: center;
+  margin-top: 20px;
+}
+
+h2 {
+  display: flex;
+  justify-content: center;
+}
+
+p {
+  color: #aa9898;
+}
+
+p strong {
```

Comment:
This is an inappropriate way of selecting its child element. It is recommended to use child combinator like this. Please see the reference below.
<img width="786" alt="Screenshot 2023-09-24 at 4 58 01 PM" src="https://github.com/myminethealmighty/microversePJ/assets/69049766/69f9be84-aada-45c1-b5c9-5d2ad4d5e231">